### ISSUE TITLE
Claim Status Tool a11y Accessibility Issue

### ISSUE THREAD

**Issue by vhenry07** [department-of-veterans-affairs/va.gov-team#36831]
## Issue Description
The listing in the Claim Status Tool contains colored dots that are not in accordance with accessibility standards. See slack[ thread here](https://dsva.slack.com/archives/C8E985R32/p1644517476221339).

---
## Tasks
- [ ] Examine design intent and develop new design to accordingly. ![Screen Shot 2022-02-10 at 13.13.07.png](https://images.zenhubusercontent.com/5ffca1cc8ef760e3db881927/346bde00-0033-4dff-b3fd-88fa2ca9b4e9)
- More from Josh:
If the intention is to bring less focus to closed items, it may be better not to have a dot icon at all
If the intention is to bring more focus to closed items, it may be better to use an icon that (a) meets 3:1 color contrast requirements and (b) doesn't rely on color for its meaning (e.g. red dot won't be interpretable to color blind users so a unique icon may be preferred)

## Acceptance Criteria
- [ ] New design forwarded to front end for implementation.
---

**Comment by Mottie** [comment]
Fixed
